app/models/region.rb

```diff
@@ -1,13 +1,17 @@
 class Region < ActiveRecord::Base
-  validates_uniqueness_of :slug
   validates_uniqueness_of :name
 
   has_many :region_organizers
   has_many :organizers, :through => :region_organizers, :source => :user
+  has_many :region_slugs, dependent: :destroy
 
   scope :active, -> { where(active: true) }
 
   def to_param
-    "#{self.id}-#{self.slug}"
+    "#{self.id}-#{self.main_slug}"
+  end
+
+  def main_slug
+    region_slugs.try(:first).try(:slug)
```

Comment:
Yes, this is indeed the lazy and unstable way of implementing it ;)

I'd add a boolean `main_slug` to `RegionSlug` and make sure it's unique for the `region`. This way the main slug can be chosen manually. 
If none is chosen, the first one returned by the database could become the main_slug as a fallback.

What do you think?